ilc/client.js

```diff
@@ -95,9 +95,13 @@ document.addEventListener('click', function (e) {
         return;
     }
 
-    const href = e.target.getAttribute('href');
-    singleSpa.navigateToUrl(href);
-    e.preventDefault();
+    const pathname = e.target.getAttribute('href');
+    const singleSpaShouldNavigateToUrl = singleSpa.checkActivityFunctions({pathname}).length > 0;
```

Comment:
It doesn't work for me: https://take.ms/PoD8a

Reply:
Fixed